Cargo.lock

```diff
@@ -2429,6 +2429,7 @@ dependencies = [
  "bytes",
  "clap",
  "criterion",
+ "crypto_box",
```

Comment:
why is this changing?

Reply:
I don't know?

Reply:
Maybe current main has this wrong after #2053